docs/en/latest/installation-guide.md

```diff
@@ -271,9 +271,13 @@ You can configure your APISIX deployment in two ways:
    apisix start -c <path to config file>
    ```
 
-APISIX will use the configurations added in this configuration file and will fall back to the default configuration if anything is not configured.
+APISIX will use the configurations added in this configuration file and will fall back to the default configuration if anything is not configured. Generally, APISIX gets installed at `/usr/local/apisix/` directory, so your configuration file will be present at `/usr/local/apisix/conf/` path.
+
+In case you get the Port binding logs, that says `Bind address already in use` when trying to run the APISIX server using `apisix start` command, it is most likely that the ports are being used by certain processes running on your machine. Try killing the process which is using the port. You can find the PID of the process that is using the process by running the `sudo netstat -tulpn` command and later using `kill <process id>` command to kill the process.
```

Comment:
```suggestion
If you are unable to start APISIX and observe `Bind address already in use` in logs, it means the ports APISIX wants to use are already occupied by other running processes. Upon reviewing the use of these processes, you may decide to kill these processes or configure APISIX to use other non-conflicting ports.
```

1. Some suggestion to reduce verbosity but you can edit again. If this is accepted, the sentence below needs to be removed as well.
2. You can add `netstat` back but note that flags can be distro / os specific (e.g. on mac `netstat: n: unknown or uninstrumented protocol`). Also `sudo` is not needed. If the audience don't have the needed privilege, they can escalate it themselves.